src/scripts/site.js

```diff
@@ -515,33 +491,12 @@ var AppInfo = {};
             define("castSenderApiLoader", [], getCastSenderApiLoader);
         }
 
-        if (self.Windows) {
-            define("bgtaskregister", ["environments/windows-uwp/bgtaskregister"], returnFirstDependency);
-            define("transfermanager", ["environments/windows-uwp/transfermanager"], returnFirstDependency);
-            define("filerepository", ["environments/windows-uwp/filerepository"], returnFirstDependency);
-        } else if ("cordova" === self.appMode) {
-            define("filerepository", ["cordova/filerepository"], returnFirstDependency);
-            define("transfermanager", ["filerepository"], returnFirstDependency);
-        } else if ("android" === self.appMode) {
-            define("transfermanager", ["cordova/transfermanager"], returnFirstDependency);
-            define("filerepository", ["cordova/filerepository"], returnFirstDependency);
-        } else {
-            define("transfermanager", [apiClientBowerPath + "/sync/transfermanager"], returnFirstDependency);
-            define("filerepository", [apiClientBowerPath + "/sync/filerepository"], returnFirstDependency);
-        }
-
-        if ("android" === self.appMode) {
-            define("localsync", ["cordova/localsync"], returnFirstDependency);
-        } else {
-            define("localsync", [apiClientBowerPath + "/sync/localsync"], returnFirstDependency);
-        }
+        define("transfermanager", [apiClientBowerPath + "/sync/transfermanager"], returnFirstDependency);
+        define("filerepository", [apiClientBowerPath + "/sync/filerepository"], returnFirstDependency);
+        define("localsync", [apiClientBowerPath + "/sync/localsync"], returnFirstDependency);
     }
 
     function init() {
-        if ("android" === self.appMode) {
-            define("nativedirectorychooser", ["cordova/nativedirectorychooser"], returnFirstDependency);
```

Comment:
I looked for NativeDirectoryChooser usage, and i found nothing. So once again, a death module. This module exposed NativeDirectoryChooser in window object. so direct usage should be looking for NativeDirectoryChooser in jf-web. Since i found no usage, i removed it.

Reply:
So most likely this dead code was for projects that were never put to open source... bye to this dead code then! Thanks for cleaning up!